### PR TITLE
fix unicode protection to allow unicode characters

### PR DESCRIPTION
**Pull request by heythisisnate**
bot review me @JonAbrams 
cc @MichaelRevell @bradherman

### CODE REVIEW

File: spec/models/concerns/subscribable_spec.rb [hunk 1]
```diff
@@ -51,12 +51,6 @@ class TestModeAuction < ActiveRecord::Base
       Reactor::Event.publish(:bid_made, target: Auction.create!(start_at: 10.minutes.from_now))
     end
 
-    it 'protects against bad unicode characters in event attributes' do
```
Comment (satyap2) [inline spec/models/concerns/subscribable_spec.rb]: Does this test fail now?

**Comment by JonAbrams** [comment]
Add `"\u{1f4a9}"` should eq `💩` to make sure astral plane characters are also supported?

File: spec/models/concerns/subscribable_spec.rb (hunk 1 above)
Comment (heythisisnate) [inline spec/models/concerns/subscribable_spec.rb]: Still passes. I just moved the test into event_spec.rb where it belongs.

**Comment by heythisisnate** [comment]
@JonAbrams good suggestion. added.

**Comment by JonAbrams** [comment]
:+1:

File: spec/models/concerns/subscribable_spec.rb (hunk 1 above)
Comment (satyap2) [inline spec/models/concerns/subscribable_spec.rb]: D'oh, ok!

**Comment by MichaelRevell** [comment]
woot!

**Pull request merged**